src/config.ts

```diff
@@ -205,3 +229,29 @@ export function getContentConfig({ BEE_API_URLS, REUPLOAD_PERIOD }: EnvironmentV
     refreshPeriod: Number(REUPLOAD_PERIOD),
   }
 }
+
+export function getFilteringConfig({ FILTERING_CONFIG_FILE }: EnvironmentVariables = {}): FilteringConfig {
+  const NO_FILTERING: FilteringConfig = { active: false, key: '', url: '', prompt: '' }
+
+  if (!FILTERING_CONFIG_FILE) {
+    //console.warn('FILTERING_CONFIG_FILE is not defined, filtering is disabled')
```

Comment:
It actually would be good to log it to the console what is the setting the gateway started with.

Reply:
Fix: https://github.com/Solar-Punk-Ltd/gateway-proxy/commit/30edb5bb54c1863cd3d726d3459dccb9ff950bbd